rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -391,6 +391,14 @@ public void registerBtcTransaction(
                 throw new RegisterBtcTransactionException("Transaction already processed");
             }
 
+            if (svpIsOngoing() && isTheSvpSpendTransaction(btcTx)) {
```

Comment:
What if we try to register the change output of the fundTx when the SVP period has ended?

Reply:
we are talking about the spend tx here, not the fund tx :)